### ISSUE TITLE
When using J+H, Released string is getting over written

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab#225]
Item 1 from when reviewing #215, 
I do not hear "Released." when using J+H. I think this is a timing issue as I see it in the A11y View.

@zepumph, is there a delay on "Released." that could be causing the "Released." string to be over written, or are the responses firing simultaneously.

If they are firing simultaneously, is it possible to delay the second one without too much trouble?

**Comment by terracoda** [comment]
Here's a screen shot from the A11y View
![Screen Shot 2019-12-31 at 11 33 58 PM](https://user-images.githubusercontent.com/1902116/71638059-39eb5d80-2c30-11ea-979a-a4ecc0b870c3.png)

**Comment by zepumph** [comment]
I am not experiencing this bug in NVDA chrome or firefox. 

>is it possible to delay the second one without too much trouble?

This is what I would think the utteranceQueue is doing, but when I poked around in there I was surprised to find that most of the timing logic was about waiting to release individual Utterance instances, as opposed to making sure that a new utterance isn't released too often. @jessegreenberg did we used to have something like an overall controller that only allows a new Utterance ever X timeperiods. For example see how `getNextUtterance` only checks based on properties on the utterance itself. Should we add something such that we don't alert too often?

**Comment by jessegreenberg** [comment]
I cannot reproduce on NVDA but I confirmed this is happening in VO.

> we used to have something like an overall controller that only allows a new Utterance ever X timeperiods.

Yes I think UtteranceQueue used to have its own delay. But I think it was collapsed into `alertStableDelay` so there was only one timing option, and so Utterance timing didn't conflict with utteranceQueue timing.

**Comment by jessegreenberg** [comment]
Confirmed that using `alertStableDelay` fixes the issue for me. @zepumph can you please review change before assigning to @terracoda to verify fix?

**Comment by zepumph** [comment]
After discussing with @jessegreenberg, we think that the above solution is best. Though it places the onus on each `Utterance` and the delay set therein to make alerts sound good. There actually isn't any over-all timing control in UtteranceQueue, instead it is up to the Utterances to manage themselves.

**Comment by zepumph** [comment]
@terracoda please review, does master seem fixed?

**Comment by terracoda** [comment]
Awesome! You got it! Closing! And thanks, i like the simple completeness of the "Released." string.
